docs/user/project_metadata.md

```diff
@@ -90,17 +90,22 @@ While the labels can be arbitrary, PyPI recognizes the ones from the
 lists below and changes the default icon from
 :fontawesome-solid-square-up-right: to a customized one.
 
+The recognition patterns are case-insensitive. Items marked with an asterisk (^*^) 
+indicate a prefix. It means that any name starting with the specified pattern 
+will be recognized.
+
 #### General URL
 
-| Icon                                      | Name          | Description               | Aliases                   |
-|:------------------------------------------|:--------------|:--------------------------|:--------------------------|
-| :fontawesome-solid-house-chimney:         | Homepage      | For the project homepage  |                           |
-| :fontawesome-solid-cloud:                 | Download      | A download link           |                           |
-| :fontawesome-solid-scroll:                | Changelog     | Changelog information     |                           |
-| :fontawesome-solid-book:                  | Documentation | Project documentation     | Docs                      |
-| :fontawesome-solid-bug:                   | Bug           | Bug/Issue report location | Issue, Tracker, Report    |
-| :fontawesome-solid-circle-dollar-to-slot: | Funding       | Sponsoring information    | Sponsor, Donation, Donate |
+| Icon                                      | Name             | Description                     | Aliases                                                                                                  |
+|:------------------------------------------|:-----------------|:--------------------------------|:---------------------------------------------------------------------------------------------------------|
+| :fontawesome-solid-house-chimney:         | Homepage         | For the project homepage        |                                                                                                          |
+| :fontawesome-solid-cloud:                 | Download         | A download link                 |                                                                                                          |
+| :fontawesome-solid-scroll:                | Changelog        | Changelog information           | Change log, Changes, Release notes, News, What's new, History                                            |
+| :fontawesome-solid-book:                  | Documentation^*^ | Project documentation | Docs^*^ , an URL pointing to [Read the Docs] domains or an URL starting with `docs.` or `documentation.` |
```

Comment:
There's a lot of these where we match based on the URL instead of the name that aren't documented. I think these should probably get all get documented in the "Hosting Platforms"/"Social Media Platforms" sections instead, with a "URL" column, not just RTD.

Reply:
Also just noticed a typo in the heading: "Social Medias Platforms"

Reply:
Good catch!